lib/bundler/index.rb

```diff
@@ -13,6 +13,9 @@ def self.build
     attr_reader :specs, :all_specs, :sources
     protected :specs, :all_specs
 
+    RUBY = "ruby"
```

Comment:
I have no attachment to using constants.  I thought bundler favored backward compatible changes and in this case, I didn't think the "new shiny" way was that much better to warrant excluding ruby 2.0 and below from the "fun".  :wink: